ani-cli

```diff
@@ -337,8 +337,8 @@ esac
 # playback & loop
 play() {
 	if printf "%s" "$ep_no" | grep -qE "^[0-9]+([^0-9][0-9]+)*$"; then
-		start=$(printf "%s" "$ep_no" | cut -d"$(printf "%s" "$ep_no" | grep -oE '[^0-9]')" -f1)
-		end=$(printf "%s" "$ep_no" | cut -d"$(printf "%s" "$ep_no" | grep -oE '[^0-9]')" -f2)
+		start=$(printf "%s" "$ep_no" | cut -d"$(printf "%s" "$ep_no" | grep -oE '[^0-9]')" -f1 2>/dev/null)
+		end=$(printf "%s" "$ep_no" | cut -d"$(printf "%s" "$ep_no" | grep -oE '[^0-9]')" -f2 2>/dev/null)
```

Comment:
Shellcheck comes from the nested subshells, and it would make a lot of sense to extract it, since the inner one is the same in both cases

Reply:
Also grep has a quiet option, does cut not?

Reply:
cat doesn't have that sadly